features/testbot/summary.feature

```diff
@@ -0,0 +1,77 @@
+@routing @basic @testbot
+Feature: Basic Routing
+
+    Background:
+        Given the profile "testbot"
+
+    @smallest
+    Scenario: Checking 
+        Given the node map
+            | a | b |  | c | d | e |
+
+        And the ways
+            | nodes |
+            | ab    |
+            | bc    |
+            | cd    |
+            | de    |
+
+        When I route I should get
+            | from | to | route          | summary  |
+            | a    | e  | ab,bc,cd,de,de | ab, bc   |
+            | e    | a  | de,cd,bc,ab,ab | de, bc   |
+            | a    | b  | ab,ab          | ab       |
+            | b    | d  | bc,cd,cd       | bc, cd   |
+
+    @smallest
+    Scenario: Check handling empty values
+        Given the node map
+            | a | b |  | c |   | d | f |
+            |   |   |  |   |   | e |
+
+        And the ways
+            | nodes | name |
+            | ab    | ab   |
+            | bc    | bc   |
+            | cd    |      |
+            | de    | de   |
+            | df    | df   |
+
+        When I route I should get
+            | from | to | route          | summary  |
+            | e    | a  | de,,bc,ab   | de, bc   |
```

Comment:
This returns a route that is named:

> de,,ab,ab 

Note no `bc`. But the s ummary is named `de,bc`. It seems the post-processing changes the name of the `bc` segment. @MoKob can you take a look at this tomorrow?